Update 'Switch Between Themes at Runtime' guide (#5264)
DevExpress/devextreme-documentation: concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md

> You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet" target="_blank">StyleSheet API</a> to switch between color swatches. In the following code, theme swatches are called `theme.custom-scheme.light.css` and `theme.custom-scheme.dark.css` and are placed in the same folder as `index.html`. The code uses SelectBox to implement the swatch selector. To switch between theme and their swatches at the same time, call [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method in the `onValueChanged` callback.

Reviewer: ```suggestion
You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet" target="_blank">StyleSheet API</a> to switch between color swatches. In the following code, theme swatches are called `theme.custom-scheme.light.css` and `theme.custom-scheme.dark.css` and are placed in the same folder as `index.html`. The code uses SelectBox to implement the swatch selector. To switch between the theme and its swatches at the same time, call [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method in the `onValueChanged` callback.
```
` at the same time` How can you switch between something 'at the same time'?

Author: I meant that you can switch between 2 themes, between 2 swatches, and between theme+swatch and another theme+swatch (the latter is the most common scenario). I know I messed up here, but I don't know how to formulate it clearly

Reviewer: Perhaps "To switch between different themes and swatches"?

        const onValueChanged = useCallback((accent) => {
            setValue(accent);
            for (let index in document.styleSheets) {
                let styleSheet = document.styleSheets[index],
                    href = styleSheet.href;
                if (href) {
                    let themeMarkerPosition = href.indexOf(themeMarker);
                    let startPosition = themeMarkerPosition + themeMarker.length,
                        endPosition = href.indexOf(".css"),
                        fileNamePart = href.substring(startPosition, endPosition);
                    if (fileNamePart.includes("custom-scheme")) {
                        styleSheet.disabled = !(
                            accent === fileNamePart.substring(fileNamePart.indexOf('.')+1)
                        );
                    }
                }
            }
        }, []);

        return (
            <React.Fragment>
                <SelectBox
                    dataSource={themeData}
                    value={value}
                    onValueChange={onValueChanged}
                />
            </React.Fragment>
        );
    }

    export default App;

---

> You can use the StyleSheet API not only to switch between swatches but to switch between themes. Refer to the following GitHub repository to learn more:

Reviewer: ```suggestion
You can use the StyleSheet API to switch between swatches and themes. Refer to the following GitHub repository for more information:
```
